### PR TITLE
Feature_test

### PR DESCRIPTION
**Pull request by FarwaMuhibZada**
#Technical_documentation_page
-  That is a technical documentation page 
-  Have a main section that contain all the articles and a navbar section which contain all the anchors tags
-  Use id selector to reference the specific article to specific links in navbar.
-  That web pages is fully responsive we use @media query that 
- Adjust our page in every screen size. this is the main feature of this webpage.
-  The navbar section located on left side when the screen size is less then 800px it will show on the top of screen.
- It has a complete readme.md file which contain the demo of my project.
- Added linters config files.
**Demo :**
Capture

### CODE REVIEW

File: index.html
Comment (ahzamir) [inline index.html:1]: - your html file element spacing is not aligned correctly. Please solve this

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @FarwaMuhibZada 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

-  Although your code passes all checks on freecodecamp website, but your code still has some issues to work on:
  - Check the comments below for more details.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
image
image

✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Technical Documentation Page and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌ Your code is clean

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: index.html
Comment (FarwaMuhibZada) [reply index.html:1]: @ahzamir 
hello sir !
I hope you are doing well . I solved and clean my html file thanks for your feedback. please review my code and approve it .

Comment (ahzamir) [reply index.html:1]: @FarwaMuhibZada 

- The identification of your spaces is very long you have to use identification of 2 spaces for spacing of your elements:
  - solve the identification of space issues **(TIP: All your child elements should be two spaces ahead of their parent element)**:
  example:
  ```
  <div>
    <h1>A title</h1>
    <form>
      <input ... />
      <input ... />
      <textarea><textarea/>
    </form>
  </div>
  ```

**Review by ahzamir — CHANGES_REQUESTED** [review]
@FarwaMuhibZada 

- Still, you did not solve the identification of element spaces in your HTML file.
  - solve the identification of space issues **(TIP: All your child elements should be two spaces ahead of their parent element)**:
  example:
  ```
  <div>
    <h1>A title</h1>
    <form>
      <input ... />
      <input ... />
      <textarea><textarea/>
    </form>
  </div>
  ```

File: index.html
Comment (FarwaMuhibZada) [reply index.html:1]: @ahzamir  I already done it sir could you check it again . I think there is problem when I push it . It doesn't  work. when  I push html file It takes 4 space could you help me I don't know  how to solve it .

Capture12

**Review by ahzamir — APPROVED** [review]
### Hi @FarwaMuhibZada 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
Screenshot 2024-05-26 134811

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WowOhMyGodGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Technical Documentation Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by FarwaMuhibZada** [comment]
@ahzamir  thank you sir

**Pull request merged**